release_notes/48-release-notes.adoc

```diff
@@ -42,7 +42,33 @@ Policy::
 
 This release adds improvements related to the following components and concepts:
 
+//ROX-20769
+[id="central-db-postgresql_15_{context}"]
+=== Central DB uses PostgreSQL 15
 
+Central DB component is using PostgreSQL 15 now, and {product-title-short} 4.8
+supports this version for external databases as well. A new installation with
+an internal database will use this version by default now. When upgrading an
+existing cluster to {product-title-short} 4.8, Central DB will do an in-place
+upgrade of the data collected so far.
+
+[NOTE]
```

Comment:
I would not have these bullets as a note (multiple lines in notes does not look the best) and have it be a sentence with bullets instead. Maybe move the part about not upgrading the database to here, right before you say "To ensure..." and make that an "important" note:

```
[IMPORTANT]
====
text
====
```

Reply:
Not quite follow what do you mean by "move the part about not upgrading the database to here"? Which part are you referring to?